content/pt-br/blog/_posts/2023-02-14-free-katacoda-kubernetes-tutorials-are-shutting-down.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: blog
+title: "Os tutoriais gratuitos do Katacoda Kubernetes estão sendo encerrados"
+date: 2023-02-14
+slug: kubernetes-katacoda-tutorials-stop-from-2023-03-31
+evergreen: true
+---
+
+**Author**: Natali Vlatko, SIG Docs Co-Chair para Kubernetes
+
+[Katacoda](https://katacoda.com/kubernetes), a popular plataforma de aprendizagem da O'Reilly que tem ajudado as pessoas a aprender tudo sobre Java, Docker, Kubernetes, Python, Go, C++ e mais, [foi desativada para uso público em junho de 2022](https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html). No entanto, os tutoriais específicos para Kubernetes, vinculados no website do Kubernetes para os usuários e contribuidores de nossos projetos, permaneceram disponíveis e ativos após esta mudança. Infelizmente, isso não será mais o caso, e os tutoriais do Katacoda para aprender o Kubernetes deixarão de funcionar após 31 de março de 2023.
+
+O Projeto Kubernetes agradece à O'Reilly Media pelos muitos anos em que apoia a comunidade por meio da plataforma de aprendizado Katacoda. Você pode ler mais sobre [a decisão de fechar o katacoda.com](https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html) no próprio website da O'Reilly. Com essa mudança, focaremos no trabalho necessário para remover os links para seus vários tutoriais. Temos uma Questão Geral para rastrear este tópico em [#33936](https://github.com/kubernetes/website/issues/33936) e [discussão no GitHub](https://github.com/kubernetes/website/discussions/38878). Também estamos interessados em pesquisar quais outras plataformas de aprendizado podem ser benéficas para a comunidade do Kubernetes, substituindo o Katacoda por uma plataforma ou um serviço que tenha uma experiência de usuário semelhante. No entanto, esta pesquisa levará tempo, por isso estamos procurando ativamente por voluntários para ajudarem neste trabalho. Se uma substituição for encontrada, ela precisará ser suportada pela liderança do Kubernetes, especificamente, SIG Contributor Experience, SIG Docs e Kubernetes Steering Committee.
```

Comment:
Some suggestions here:
* How about not translating "issue" in "general issue"? I suggest using "uma issue geral do GitHub".
* I think we could translate "Kubernetes Steering Comittee". Suggestion: "Comitê de Gestão do Kubernetes".

Reply:
Sinceramente, eu prefiro sim o "issue" ao inves de traduzir para "problema" ou "questão" ou algo similar. Mas fico a disposição para o que achar melhor.

Reply:
Estou colocando _issue_ (itálico). Concordo ser melhor por ser um conceito conhecido e amplamento utilizado.